aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -50,6 +50,10 @@ pub enum EntryFunctionCall {
         recipient_address: AccountAddress,
     },
 
+    AccountRevokeSignerCapability {
```

Comment:
Wut? Any reason these are included?

Reply:
it's auto generated for whatever reason, I asked in #smart-contracts-eng